target/linux/mediatek/dts/mt7986a-xiaomi-redmi-router-ax6000.dts

```diff
@@ -202,6 +214,35 @@
 	};
 };
 
+&spi1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&spi_led_pins>;
+	cs-gpios = <0>, <0>;
```

Comment:
Does it really need a cs-gpios property?

Reply:
Of course not, copied it from spi0 by mistake, I will delete it, thanks